pipeline-runner/R/qc-3-filter_high_mito.R

```diff
@@ -89,13 +89,13 @@ filter_high_mito <- function(scdata, config, sample_id, cells_id, task_name = "m
 }
 
 
-# The most uses values in MT-content are between [0.1, 0.2]. There are not too much literature about how to compute
-# a threshold.
-# --> Absolute threshold: In order to be not too extrictive the threshold is set to 0.1
 generate_default_values_mitochondrialContent <- function(scdata, config) {
   if (config$filterSettings$method == "absolute_threshold") {
-    # HARDCODE
-    threshold <- 0.1
+    is.out <- scuttle::isOutlier(scdata$percent.mt, type = "higher")
+
+    threshold <- ifelse(!sum(is.out),
+                        max(scdata$percent.mt),
+                        min(scdata$percent.mt[is.out])) / 100
```

Comment:
If I understand correctly the idea is that we use the min outlier or the max scdata$percent value? If we do that we won't be filtering anything if there are no outliers, right?

I saw something similar in the [OSCA handbook](http://bioconductor.org/books/3.14/OSCA.basic/quality-control.html#common-choices-of-qc-metrics), is that what you implemented here?